### ISSUE TITLE
Lora max/min weight adjust parameters??

### ISSUE THREAD

**Issue by Echolink50** [lllyasviel/Fooocus#1249]
I am trying to find the config file that controls the Lora sliders min/max value. Some loras use higher values than 2. Thanks

**Comment by nftcharm** [comment]
Hello there! I am currently looking into the javascript code to answer your question. Foocus is build with svelte, so I guess it's in some compiled JS.

In the meanwhile, I would like to know, which version of stable defusion are we using here with Fooocus? I downloaded Foococus freshly 3 days ago. I am trying to use Lora's and it seems like I can only use Lora's with version SDXL 1.0?

**Comment by Echolink50** [comment]
Yes, Fooocus only works with SDXL models, loras, controlnets, etc. I usually use JuggernautXLv6. The creators realistic model is good too. Thanks for looking into it. If you find the answer post up.

**Comment by Echolink50** [comment]
Yes, Fooocus only works with SDXL models, loras, controlnets, etc. I usually use JuggernautXLv6. The creators realistic model is good too. There's a config file so I just put my Auto1111 model directory in there. I also have my Comfyui install pointed at the Auto1111 models. Saves some space. I use all three. I just installed Fooocus mainly for inpainting and outpainting. Thanks for looking into the Lora weight max thing. If you find the answer post up.

**Comment by mzisa** [comment]
Just in case, now you may perform this change, without altering the code, by modifying the config.txt file using these keys:

"default_loras_min_weight"
"default_loras_max_weight"
